Tests/SwiftLintFrameworkTests/DiscouragedDirectInitRuleTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  DiscouragedDirectInitRuleTests.swift
+//  SwiftLint
+//
+//  Created by Ornithologist Coder on 8/1/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SwiftLintFramework
+import XCTest
+
+class DiscouragedDirectInitRuleTests: XCTestCase {
+    let baseDescription = DiscouragedDirectInitRule.description
+
+    func testDiscouragedDirectInitRuleWithDefaultConfiguration() {
```

Comment:
could you remove the `Rule` from the method names just to keep consistent with other tests?